Project.toml

```diff
@@ -6,9 +6,14 @@ version = "0.1.5"
 [deps]
 Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
-PythonCall = "6099a3de-0909-46bc-b1f4-468b9a2dfc0d"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
+[weakdeps]
+PythonCall = "6099a3de-0909-46bc-b1f4-468b9a2dfc0d"
+
+[extensions]
+AwkwardPythonCallExt = "PythonCall"
```

Comment:
@Moelf - we have started with it as an extension, but it did not work as expected. @jpivarski and I discussed and agreed that the dependency should be mandatory since the package is intended to be used as a connection between Julia and Python - e.g. not as a replacement to numerous Julia arrays implementation :-).

Reply:
Awkward provides a nice framework for organization schema, I think there's pure-Julia usage case, for example, for UnROOT.jl's schema system.

Reply:
tests pass now I wonder what's the workflow that didn't work -- happy to triage that

Reply:
I don't see `AwkwardPythonCallExt.jl` in code coverage anymore. I think, because of that its percentage is below the required :-(

The 1.8 and 1.9 were required workflows - hat is easy to fix.

Reply:
https://app.codecov.io/gh/JuliaHEP/AwkwardArray.jl/pull/111/blob/ext/AwkwardPythonCallExt.jl

looks like this works.

I wonder what's happening in Base.

Reply:
yes, I can see python tests run successfully!